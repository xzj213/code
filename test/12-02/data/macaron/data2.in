3
2234 3919 789
1026 1631 2487
1129 2292 2922
