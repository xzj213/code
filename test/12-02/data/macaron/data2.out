39 291 657 1423 2553 4067 
